packages/logger/src/utils/format.ts

```diff
@@ -87,7 +88,6 @@ function humanReadableTemplateFn(_info: {[key: string]: any; level: string; mess
   str += `[${infoString}] ${info.level.padStart(infoPad)}: ${info.message}`;
 
   if (info.context !== undefined) str += " " + logCtxToString(info.context);
-  if (info.error !== undefined) str += " " + logCtxToString(info.error);
-
+  if (info.error !== undefined) str += (info.error instanceof LodestarError ? " " : " - ") + logCtxToString(info.error);
```

Comment:
We barely use `LodestarError` but it is handled in the same way as `info.context` in `logCtxToString` and it is better to just use a white space as separator for this
https://github.com/ChainSafe/lodestar/blob/f5e2c3a40c9a01a29a4283d9f17a901ede413068/packages/logger/src/utils/json.ts#L105